src/scenes/ProgressReports/EditForm/ProgressReportSidebar.tsx

```diff
@@ -29,6 +30,11 @@ export const ProgressReportSidebar = ({ report }: ReportProp) => {
       >
         Instructions
       </Button>
+      <Box mb={1}>
```

Comment:
This leaves zero spacing between the instructions button.

![Screenshot 2023-02-01 at 3 08 02 PM](https://user-images.githubusercontent.com/932566/216163592-03bab8fd-bb74-437d-afc5-4d642626dfe4.png)

Also the Box is still rendered even without the chip - it should be moved into the conditional.